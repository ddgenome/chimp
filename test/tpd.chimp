mechanism "tpd.mech"
parameter "tpd.par"
task "tpd.task"
mechanism "tpd.explicit.mech"
parameter "tpd.par"
task "tpd.explicit.task"
