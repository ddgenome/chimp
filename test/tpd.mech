# TPD mechanism with nearest-neighbor interactions
@A + 4 [@A, @B]       -> k_arrhenius(A_4, E_4) A + @;
@A + 3 [@A, @B] +   @ -> k_arrhenius(A_3, E_3) A + 2 @;
@A + 2 [@A, @B] + 2 @ -> k_arrhenius(A_2, E_2) A + 3 @;
@A +   [@A, @B] + 3 @ -> k_arrhenius(A_1, E_1) A + 4 @;
@A +              4 @ -> k_arrhenius(A_0, E_0) A + 5 @;

# reaction to make sure step is short enough
X -> k_constant(1.0e-5) X;
