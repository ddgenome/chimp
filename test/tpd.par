# TPD with adsorbate-adsorbate interactions
A_4	1.0e11
E_4	5.0e4
A_3	1.0e11
E_3	6.0e4
A_2	1.0e11
E_2	7.0e4
A_1	1.0e11
E_1	9.0e4
A_0	1.0e11
E_0	9.0e4
