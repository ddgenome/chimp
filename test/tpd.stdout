chimp 0.1
Copyright (C) 2004 David Dooling <banjo@users.sourceforge.net>

There is ABSOLUTELY NO WARRANTY; see file COPYING for details.
This is free software, and you are welcome to redistribute it
under certain conditions; see file COPYING for details.

Report bugs to http://sourceforge.net/projects/chimp/.
chimp: performing task tpd.par... completed; 0u 0s 0w
chimp: performing task tpd... completed; 2.96u 0.02s 2.98w
chimp: performing task tpd.par... completed; 0u 0s 0w
chimp: performing task tpd_explicit... completed; 3.05u 0s 3.05w
